recipes/openssl/1.x.x/conanfile.py

```diff
@@ -357,6 +357,10 @@ def _tool(self, env_name, apple_name):
             return getattr(tools.XCRun(self.settings), apple_name)
         return None
 
+    def _patch_configure(self):
```

Comment:
It would be great to have a similar comment note about why this is needed, maybe even with some reference,  as the `_patch_makefile_org` has one. Can this be please added?

Reply:
done